src/main/java/dan200/computercraft/core/computer/ComputerThread.java

```diff
@@ -139,7 +140,7 @@ public void run() {
                                     // Step 3: abandon
                                     if( worker.isAlive() )
                                     {
-                                        //System.out.println( "computercraft: Warning! Failed to abort Computer " + computercraft.getDescription() + ". Dangling lua thread could cause errors." );
+                                        ComputerCraft.log.warn( "Failed to abort Computer " + computer.getID() + ". Dangling lua thread could cause errors." );
```

Comment:
Until we've fixed cases where this can happen, could we leave this line commented out? Otherwise you're introducing a vector for a malicious user to flood the server's log.